### PR TITLE
Save file

### PR DESCRIPTION
**Pull request by anrui811**
@mxjl620

### CODE REVIEW

**Comment by kimimj** [comment]
LGTM ,but i need time to do some test later

**Comment by anrui811** [comment]
i have updated @kimimj

File: crawl.py [hunk 1]
```diff
@@ -43,12 +66,13 @@ def startCrawl(urlList):
 		urlList.remove(url)
 		if url in crawledUrls:
 			continue
+		writeData(url)
 		crawledUrls.append(url)
```
Comment (kimimj) [inline crawl.py]: ANOTHER INDENTATION PROBLEM

File: crawl.py [hunk 2]
```diff
@@ -28,14 +29,36 @@ def getURL(page):
...
-			tempList.append(tempUrl)
-	print tempList
+		if item.find("..") != -1:
+			continue
+		if item.find("xjtu") != -1:
+			if item.find("http://") != -1:
+				tempList.append(item)
+			else:
+				tempUrl = baseUrl + str(item)
+				tempList.append(tempUrl)
```
Comment (kimimj) [inline crawl.py:39]: PLEASE CHECK HERE

**Comment by kimimj** [comment]
@anrui811 IN MY Opinion you need be more careful next time

File: crawl.py [hunk 3]
```diff
@@ -28,14 +29,36 @@ def getURL(page):
...
-			tempList.append(item)
-		else:
-			tempUrl = baseUrl + str(item)
-			tempList.append(tempUrl)
-	print tempList
+		if item.find("..") != -1:
+			continue
+		if item.find("xjtu") != -1:
+			if item.find("http://") != -1:
+				tempList.append(item)
```
Comment (kimimj) [inline crawl.py:36]: and here

File: crawl.py [hunk 4]
```diff
@@ -43,12 +66,13 @@ def startCrawl(urlList):
 		urlList.remove(url)
 		if url in crawledUrls:
 			continue
+		writeData(url)
 		crawledUrls.append(url)
 		print url,len(urlList)
 		temp = getURL(getPage(url))
-		print temp
 		temp.extend(urlList)
```
Comment (kimimj) [inline crawl.py:73]: and here

**Comment by kimimj** [comment]
🀄︎LGTM

**Comment by mxjl620** [comment]
🚬 Actually it's a DF,you can check it ,if you can not get it ,ask me on Skype ,i will explain to you

**Pull request merged**